packages/wdio-browserstack-service/src/service.ts

```diff
@@ -97,6 +99,17 @@ export default class BrowserstackService implements Services.ServiceInstance {
         this._scenariosThatRan = []
 
         if (this._observability && this._browser) {
+            const BSTestOpsPatcher = new logPatcher({})
+            // @ts-ignore
+            // eslint-disable-next-line no-global-assign
+            console = {}
+            Object.keys(consoleHolder).forEach(method => {
+                // @ts-ignore
+                console[method] = (...args) => {
+                    // @ts-ignore
+                    BSTestOpsPatcher[method](...args)
+                }
+            })
```

Comment:
This can potentially impact other services. I suggest to keep the console behavior and patch it so that it also calls, e.g.
```suggestion
            Object.keys(consoleHolder).forEach(method => {
                console[method] = (...args) => {
                    process.stdout.write(d + '\n')
                    BSTestOpsPatcher[method](...args)
                }
            })
```

Also, let's make sure to type this correctly

Reply:
Have resolved the type issues in [this commit](https://github.com/webdriverio/webdriverio/pull/11098/commits/9cd70dd11c62da864475adcd1914ba336ded8de7). Also reg. this `process.stdout.write(d + '\n')`, we are already doing it in [logPatcher](https://github.com/webdriverio/webdriverio/blob/ac14908cdbc1795284d61ab3cc2454e2eced53e5/packages/wdio-browserstack-service/src/logPatcher.ts#L14), where all console logs will get logged